app.js

```diff
@@ -1,5 +1,37 @@
-// import functions and grab DOM elements
+/* eslint-disable quotes */
+let y = Math.floor(Math.random() * 20 + 1); 
+   
+// counting the number of guesses 
+// made for correct Guess 
+let number = 1; 
```

Comment:
same for number as a variable, which number are we looking for here? Maybe renaming this as numberOfGuesses could work better.